src/Cli/dotnet/commands/dotnet-package/PackageCommandParser.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.CommandLine;
+
+namespace Microsoft.DotNet.Cli
+{
+    internal class PackageCommandParser
+    {
+        public static readonly string DocsLink = "https://aka.ms/dotnet-package";
```

Comment:
Seemed there're no other references, and no need to update, maybe we could let it `private` and be a `const`

```suggestion
        private const string DocsLink = "https://aka.ms/dotnet-package";
```